File: handin2/Code/readbodyfat.R
#
#  Reads the body fat data set from the bodyfat.mat file.
#

load( 'bodyfat.RData' )

ridx <- sample( 1:dim( data )[1], dim( data )[1] )

test <- data[ridx[201:length( ridx )],]
train <- data[ridx[1:200],]

File: handin2/src/readbodyfat.m
function [train, test] = readbodyfat
% [train, test] = readbodyfat
%
%  Reads the body fat data set from the bodyfat.mat file and returns 
%  appropriate training and test sets.

load bodyfat
ridx = randperm(size(data,1));
test = data(ridx(201:end),:);
train = data(ridx(1:200),:);

end

